content/en/docs/reference/scheduling/policies.md

```diff
@@ -16,94 +16,13 @@ You can set a scheduling policy by running
-  {{< glossary_tooltip text="PVCs" term_id="persistent-volume-claim" >}}.
-
-## Priorities
-
-The following *priorities* implement scoring:
-
-- `SelectorSpreadPriority`: Spreads Pods across hosts, considering Pods that
-   belong to the same {{< glossary_tooltip text="Service" term_id="service" >}},
-   {{< glossary_tooltip term_id="statefulset" >}} or
-   {{< glossary_tooltip term_id="replica-set" >}}.
-
-- `InterPodAffinityPriority`: Implements preferred
-  [inter pod affininity and antiaffinity](/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity).
-
-- `LeastRequestedPriority`: Favors nodes with fewer requested resources. In other
-  words, the more Pods that are placed on a Node, and the more resources those
-  Pods use, the lower the ranking this policy will give.
-
-- `MostRequestedPriority`: Favors nodes with most requested resources. This policy
-  will fit the scheduled Pods onto the smallest number of Nodes needed to run your
-  overall set of workloads.
-
-- `RequestedToCapacityRatioPriority`: Creates a requestedToCapacity based ResourceAllocationPriority using default resource scoring function shape.
-
-- `BalancedResourceAllocation`: Favors nodes with balanced resource usage.
-
-- `NodePreferAvoidPodsPriority`: Prioritizes nodes according to the node annotation
-  `scheduler.alpha.kubernetes.io/preferAvoidPods`. You can use this to hint that
-  two different Pods shouldn't run on the same Node.
-
-- `NodeAffinityPriority`: Prioritizes nodes according to node affinity scheduling
-   preferences indicated in PreferredDuringSchedulingIgnoredDuringExecution.
-   You can read more about this in [Assigning Pods to Nodes](/docs/concepts/scheduling-eviction/assign-pod-node/).
-
-- `TaintTolerationPriority`: Prepares the priority list for all the nodes, based on
-  the number of intolerable taints on the node. This policy adjusts a node's rank
-  taking that list into account.
-
-- `ImageLocalityPriority`: Favors nodes that already have the
-  {{< glossary_tooltip text="container images" term_id="image" >}} for that
-  Pod cached locally.
-
-- `ServiceSpreadingPriority`: For a given Service, this policy aims to make sure that
-  the Pods for the Service run on different nodes. It favours scheduling onto nodes
-  that don't have Pods for the service already assigned there. The overall outcome is
-  that the Service becomes more resilient to a single Node failure.
-
-- `EqualPriority`: Gives an equal weight of one to all nodes.
-
-- `EvenPodsSpreadPriority`: Implements preferred
-  [pod topology spread constraints](/docs/concepts/workloads/pods/pod-topology-spread-constraints/).
+{{< warning >}}
+This scheduling policy is removed entirely, associated flags `policy-config-file`, `policy-configmap`, `policy-configmap-namespace` and `use-legacy-policy-config` are also removed. Instead, use the [Scheduler Configuration](/docs/reference/scheduling/config/) to achieve similar behavior.
```

Comment:
is not longer supported.


Also remove lines 14 to 17. Not sure if also 9 to 12.

Reply:
We just removed `how to do it` and I think we can keep `what it is`, since it's a short description. Just from my side.

Reply:
/lgtm

Although I'll leave it to the docs reviewer to confirm if this is a good strategy.